domains/AidzLT.json

```
@@ -0,0 +1,9 @@
+{
+    "owner": {
+        "username": "aidzlt",
+        "email": "aidzlt0001@gmail.com"
+    },
+    "record": {
+        "CNAME": "aidzlt.github.io/"
```

please remove the custom domain from your github repo, you can add it back after the PR has been merged
CNAME can't contain slashes
```suggestion
        "CNAME": "aidzlt.github.io"
```